Sources/Fakery/Generators/Address.swift

```diff
@@ -1,5 +1,10 @@
 import Foundation
 
+public struct FakeryCoordinates {
```

Comment:
How about using `CLLocationCoordinate2D` instead of this struct?

Reply:
I thought about that too. It requires a MapKit import.
If you think it won't be a problem I'll change it.

Reply:
I think it should be fine seeing that it's not a third-party dependency and `Fakery` is a framework for unit tests.